regression/apps/vsock/vsock_client.c

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MPL-2.0
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/socket.h>
+#include <linux/vm_sockets.h>
+#include <unistd.h>
+#include <string.h>
+
+#define PORT 1234
+
+int main()
+{
+	int sock;
+	char *hello = "echo 'Hello from host'\n";
```

Comment:
Since there may be multiple commands, I think we should define these commands in a seperate file, for example, `vsock_commands`, then, `vsock_client` will read one line from `vsock_commands` one time to execute until all commands are executed.